poly/commands.go

```diff
@@ -34,6 +35,8 @@ Tim
 
 ******************************************************************************/
 
+var errIllegalFlag error = errors.New("the '-i' flag can only be used with pipes")
```

Comment:
`errIllegalFlag` should be `errIllegalInputFlag` just in case there is ever any other illegal flags we should be throwing errors for.